opendevin/runtime/client/runtime.py

```diff
@@ -127,7 +136,7 @@ async def _init_container(
                     f'--working-dir {sandbox_workspace_dir} '
                     f'--plugins {plugin_names}'
                 ),
-                network_mode='host',
+                **network_kwargs,
```

Comment:
Kwargs reduce legibility a bit, so it might be better to just use arguments?

Reply:
I guess the tricky thing is that we have two different key arguments for different scenarios, which might complicate things a bit.

Plus, we have a separate workflow dedicated for running `test_runtime` (it needs more space for docker images?), so i think we might leave #2993 as is (unless we want to run `test_runtime` together with other testcases).

Reply:
Ah, got it. Thanks!

Reply:
In terms of the different key arguments, it seems that we could pass in `None` when arguments are not used, but doing it with kwargs is not a huge deal either.

Reply:
Just fixed kwargs in the latest commit!